packages/experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -114,4 +114,19 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * Helper for the drag and drop
    */
   dragDropHelper?: IDragDropHelper;
+
+  /**
+   * Callback for when items need to be converted to a string for a drag action
+   */
+  getSerializedItems?: (items: T[]) => string;
```

Comment:
might want to make the callback name more specific to drag/drop ? Same for the deserailized one?